battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java

```diff
@@ -130,11 +120,6 @@ public void start(final Stage primaryStage) {
 
     @Override
     public void run() {
-        try {
-            shipsPlaced.acquire();
-        } catch (InterruptedException e) {
-            output.send(e.getMessage());
-        }
         while (true) {
```

Comment:
while(true) - there should be an ending condition

Reply:
ok